templates/taxi/manufacturer_list.html

```diff
@@ -23,10 +25,23 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a
+                    href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}"
+                    style="color: red;"
+                >Delete</a>
```

Comment:
Consider using a form with the `POST` method for the delete action to prevent unintended actions from being triggered by simple URL visits.